controllers/datafoundation.go

```diff
@@ -70,6 +70,10 @@ func DFAddToScheme(scheme *runtime.Scheme) {
 	utilruntime.Must(ocsv1.AddToScheme(scheme))
 }
 
+func dfIsReadyToBeRemoved(r *ManagedFusionOfferingReconciler, offering *v1alpha1.ManagedFusionOffering) (bool, error) {
```

Comment:
- Better have verb at the start in func sig, `dfIs` vs `isDf`

Reply:
the `df` prefix  is our internal marker for all offering-related callbacks, this is going to change when we separate the DF code into its own package

Reply:
> this is going to change
- ack, the current functions always read as that something is ready to be removed 😅 
- fine for now, if this is slated for a refactor